ShopperInsights/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
we have one `gitignore` file for the entire project - I don't think we need the module-specific one